src/firebase.js

```diff
@@ -25,6 +26,24 @@ if (window.location.hostname === "localhost") {
   db.useEmulator("localhost", 8080);
 }
 
+// TODO: Can store variables for the nested pieces of a record https://firebase.google.com/docs/firestore/data-model#references
+
+// Attempts to retrieve the data in /{collectionName}/{study_id}/participants/{participant_id}
+// Will return false if participant isn't valid
+const validateParticipant = async (participantId, studyId) => {
```

Comment:
For online experiments there are two common situations, when participants are preregistered (small clinical trials) or when participants are not (large online prolific experiments). Maybe we should have a setting variable that controls for that?

Reply:
#147